lib/feedsme/index.js

```diff
@@ -478,6 +478,9 @@ Feedsme.prototype.increment = function inc({ publish, strategy, payload, head, p
       defaultInc();
       break;
   }
+
+  this.log.info(`Incrementing ${name} from (${prevVersion}|${releaseVersion}|${previousReleaseVersion}) to ${version} using ${strategy}.`);
```

Comment:
so technically we aren't ALWAYS incrementing. on publish we are but we are mostly prepping the payload being promoted for other env. we should still compare prevVersion to version but in standard case for anything non DEV, releaseVersion should always equal version.

We should log the latest dist-tag version as well as these two versions.